safe/plugin.py

```diff
@@ -801,7 +801,7 @@ def show_welcome_message(self):
         # Set previous_version to the current inasafe_version
         set_setting('previous_version', inasafe_version)
 
-        if setting('always_show_welcome_message'):
+        if setting('always_show_welcome_message', expected_type=bool):
```

Comment:
in the setting helper, you can add automatically the expected_type based on the type in default_settings.py and not having to specify the type here

Might be good to do the same with default value

Reply:
ah, brilliant.

I will add a ticket for it. Not sure if I had time today. I still don't understand why this code is working.